packages/maumau-server/src/game/reducer.ts

```diff
@@ -51,12 +51,17 @@ export function reducer(state: State, action: Action): State {
       const newHand = player.hand.filter((card) => !card.isEqual(action.payload));
       players[playersTurnIndex].hand = newHand;
 
+      const nextPlayerIndex = (playersTurnIndex + 2) % players.length;
+      const nextPlayer = players[nextPlayerIndex];
+
       return {
         ...state,
         players,
         stack: [...stack, action.payload],
         playersTurnIndex: (playersTurnIndex + 2) % players.length,
         hasDrawnCard: false,
+        nextSuit: null,
+        gameEnded: newHand.length === 0 && nextPlayer.id !== player.id,
```

Comment:
We could also just remove `&& nextPlayer.id !== player.id,` and it would be a similar implementation 🤙